src/sveltekit/sdk-setup.ts

```diff
@@ -59,19 +62,25 @@ export async function createOrMergeSvelteKitFiles(
 
   const { dsn } = projectInfo;
 
+  Sentry.setTag(
+    'server-hooks-file-strategy',
```

Comment:
I think the labels for these tags are swapped.

Reply:
Good catch, thanks!